### PR TITLE
Header & footer implementation

### PR DESCRIPTION
**Pull request by ashotvdovychenko**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -16,6 +16,31 @@ body {
...
+    background: transparent;
+    border: 0;
+}
+
+.nav-burger-line {
+    width: 30px;
+    height: 3px;
+    background-color: black;
+    margin: 3px 0;
+}
```
Comment (neodavis) [inline src/style.css:31]: Could we rewrite while `.nav-burger-toggle` and partly `.nav-burger-line` selector rules with bootstrap?)

**Review by neodavis — CHANGES_REQUESTED** [review]
PR is fine, but please, resolve following comment)

File: src/style.css (hunk 1 above)
Comment (ashotvdovychenko) [reply src/style.css:24]: Pushed the fix.

**Review by neodavis — APPROVED** [review]

**Review by Pagendera — APPROVED** [review]
Well done, good job

**Pull request merged**